docs/installation_configuration_guide/ScenarioDeploymentConfiguration.md

```diff
@@ -5,16 +5,13 @@ sidebar_position: 3
 
 # Scenario Deployment configuration
 
-In order to deploy scenario on the given [Engine](../about/engines), you need to configure the deployment.
+Deployment of a scenario onto the [Engine](../about/engines) is managed by Designer's extension called [Deployment Manager](../about/GLOSSARY#deployment-manager).
```

Comment:
by "a" Designer's extension?

Reply:
I asked chatGPT, who suggested "the" because the Designer is a well-defined entity.

Reply:
Yeah, the looks better